app.json

```diff
@@ -19,17 +16,16 @@
     "updates": {
       "fallbackToCacheTimeout": 0
     },
-    "assetBundlePatterns": [
-      "**/*"
-    ],
+    "assetBundlePatterns": ["**/*"],
     "ios": {
       "bundleIdentifier": "tools.disciple.app",
       "supportsTablet": true
     },
     "android": {
       "package": "tools.disciple.app",
+      "googleServicesFile": "./google-services.json",
```

Comment:
It looks like this may be implemented for Android only?  (We're supporting iOS now also)

Reply:
works for iOS too 😄